addon/reducers/frost-select.js

```diff
@@ -219,6 +219,9 @@ export default function reducer (state, action) {
       nextState = select(state, action.itemIndex)
       break
     case SELECT_HOVER:
+      if (state.hoveredItem === null) {
+        return state
+      }
```

Comment:
The problem is that `nextState.lastAction` is set outside the switch statement, and `lastAction` is used to determine if onChange should be called, which gets the values to pass up in `getValues`, which is where the bug really is:
```
getValues () {
    const state = this.get('reduxStore').getState()
    return [state.baseItems[state.selectedItem].value]
  }
```
Do you want to just let the state progress as usual but rewrite that function to be:
```
  getValues () {
    const state = this.get('reduxStore').getState()
    const values = state.selectedItem !== null ? [state.baseItems[state.selectedItem].value] : null
    return values
  }
```
?